tests/NgTests/DnxCatalogCollectorTests.cs

```diff
@@ -88,5 +90,40 @@ public class DnxCatalogCollectorTests
             Assert.Null(otherPackageNuspec.Key);
             Assert.Null(otherPackageNupkg.Key);
         }
+
+        [Theory]
+        [InlineData("1.2.0")]
+        [InlineData("1.2")]
+        [Description("Test the dnxmarker save and delete scenarios.")]
+        public async Task DnxMarkerTestVersion(string version)
+        {
+            string id = "testid";
+            // Arrange
+            var catalogToDnxStorage = new MemoryStorage();
+            var catalogToDnxStorageFactory = new TestStorageFactory(name => catalogToDnxStorage.WithName(name));
+            DnxMaker marker = new DnxMaker(catalogToDnxStorageFactory);
+
+            var nupkg = new MemoryStream();
+            StreamWriter writer = new StreamWriter(nupkg);
+            writer.Write("nupkg data");
+            writer.Flush();
+            nupkg.Position = 0;
+
+            var dnxEntry = await marker.AddPackage(nupkg, "nuspec data", id, version, CancellationToken.None);
+            var ns = dnxEntry.Nuspec;
+            var npkg = dnxEntry.Nupkg;
+
+            string normalizedVer = DnxMaker.Normalize(version);
```

Comment:
If you don't remove `DnxMaker.Normalize`, I think you should call `NuGetVersion.Parse` or `NuGetVersion.TryParse` directly for this test case--if there's a bug in `DnxMaker.Normalize` this test case will not catch it.